### DISCUSSION TITLE
v6.2.8

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#509] [公告]
*好好好，一发稳定版 BUG 就全都找到了是吧（*

## 错误修复

* 修复用户设置 Token 时添加非 ASCII 字符时出现 Headers 请求错误
* 修复 qBittorrent 封禁列表全量修复失败的问题，导致封禁操作失败
* 修复 banWave 过程中出现错误时，异常被静默吞掉的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.2.8'>v6.2.8</a>.</em>

**Comment by Swzer** [comment]
625之后的MACOS版本都用过了,GUI版一直都弹跳图标,打不开.只能在nogui下用了,M3 pro芯片,系统版本15.1 beat,有其他mac用户正常的吗?如果只有我一个的话,那我再找找自己问题...

**Reply by Ghost-chu** [reply]
macOS 版本是未测试过的，因为开发组没有 macOS 设备。

**Reply by SlenderData** [reply]
我也有同样的问题，建议使用 Docker 版本。

**Reply by Swzer** [reply]
> macOS 版本是未测试过的，因为开发组没有 macOS 设备。

好的,谢谢回复.
